uswds/uswds
# USWDS  - Build: Modify build tasks to get more reliable output on release.

## Description
Resolves #4072. The task `make-tmp-directory` in `release.js` was being started before `build` task was finished. This resulted in incomplete ZIP files of release builds.

- Modifies tasks in`svg-sprite.js` so the streams are returned after they finish
- Adds `svg-sprite` as parallel task

## Additional information
Steps to test:
- Pull branch
- Run `npx gulp release`
- Verify contents of ZIP file, specifically `scss/utilities/palettes` as well as `img/` and `img/usa-icons`
  Previously, builds _looked_ successful but img directory would be empty or more nested SCSS folders would be empty. Aside from JS directory being missing.

Before you hit Submit, make sure you’ve done whichever of these applies to you:
